src/components/Modal/Modal.vue

```diff
@@ -189,20 +189,19 @@
 					@mousedown.self="close">
 					<!-- Navigation button -->
 					<transition name="fade-visibility">
-						<a v-show="hasPrevious && !clearView"
+						<Actions v-show="hasPrevious && !clearView"
 							class="prev"
-							href="#"
 							:class="{
 								invisible: clearView || !hasPrevious
 							}"
 							@click.prevent.stop="previous">
-							<span class="icon-previous">
-								<ChevronLeft :size="40" title="" decorative />
-								<span class="hidden-visually">
-									{{ t('Previous') }}
-								</span>
-							</span>
-						</a>
+							<ActionButton aria-label="Previous item">
```

Comment:
I would propose to use an icon-only `Button` instead of `Actions` + `ActionButton`, see https://nextcloud-vue-components.netlify.app/#/Components/Button.

Reply:
Fixed